packages/components/src/modal/Modal.scss

```diff
@@ -0,0 +1,100 @@
+@import '../../scss/custom.scss';
+
+@keyframes slideInFromTop {
+  0% {
+    transform: translateY(-50px);
+  }
+  100% {
+    transform: translateY(0);
+  }
+}
+
+@keyframes backdropFadeIn {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+.modal {
```

Comment:
you probably don't need any or hardly any of this css, it's already part of bootstrap